modin/core/storage_formats/pandas/parsers.py

```diff
@@ -116,7 +116,12 @@ def _split_result_for_readers(axis, num_splits, df):  # pragma: no cover
         A list of pandas DataFrames.
     """
     splits = split_result_of_axis_func_pandas(
-        axis, num_splits, df, min_block_size=MinPartitionSize.get()
+        axis,
+        num_splits,
+        df,
+        min_block_size=(
+            MinRowPartitionSize.get() if axis == 0 else MinColumnPartitionSize.get()
+        ),
```

Comment:
Looks more natural, doesn't it?

```suggestion
        min_block_sizes=MinPartitionSizes.get(),
```

Reply:
The function has the value of the axis parameter; the function itself will select the desired value from the tuple using it.